desk/app/groups.hoon

```diff
@@ -1827,9 +1830,16 @@
       =/  =action:g  [flag now.bowl %cordon %shut %del-ships %ask ships]
       (poke-host /rescind act:mar:g !>(action))
     ++  get-preview
-      %^  subscribe  (welp ga-area /preview)
-        [p.flag dap.bowl]
-      /groups/(scot %p p.flag)/[q.flag]/preview
+      |=  delay=?
+      =/  =wire  (welp ga-area /preview)
+      =/  =dock  [p.flag dap.bowl]
+      =/  =path  /groups/(scot %p p.flag)/[q.flag]/preview
+      ^+  cor
+      =^  caz=(list card)  subs
+        (~(unsubscribe s [subs bowl]) wire dock)
+      =^  crds=(list card)  subs
+        (~(subscribe s [subs bowl]) wire dock path delay)
+      (emil (welp caz crds))
```

Comment:
I don't think we need any delays on these, since these are "single read" subscriptions. (Using remote scry for these should go on the groups agent refactor list.)